pom.xml

```diff
@@ -35,7 +37,7 @@
         <artifactId>maven-plugin</artifactId>
         <version>2.7.1</version>
         <optional>true</optional>
-        <exclusions>
+        <exclusions> <!-- TODO use 3.0 -->
```

Comment:
Just upgrade it to 3.1 in a separate PR?

Reply:
Maybe. Up to maintainer I guess.

Reply:
I agree to use maven-plugin 3.0+ to simplify the dependency.
I'll do that before releasing the next version.

I prefer to use the least version and plan to use 3.0, but let me know if 3.1 (or 3.1.2, the latest one) is better than 3.0.

Reply:
I would recommend the latest.

Reply:
Ok, I'll update to the latest.